internal/gcp/tags.go

```diff
@@ -178,3 +180,45 @@ func (m *tagsManager) GetProjectInfo(ctx context.Context, projectID string) (*re
 	m.cache.Set(cacheKey, project, tagCacheDuration)
 	return project, nil
 }
+
+func (m *tagsManager) DeleteValue(ctx context.Context, projectID string, key string, value string) error {
+
+	req := &resourcemanagerpb.DeleteTagValueRequest{
+		Name: value,
+	}
+
+	op, err := m.valuesClient.DeleteTagValue(ctx, req)
+	if err != nil {
+		return nil // Tag value already in use or deleted, consider this a success
```

Comment:
Is there any way to differentiate in use/deleted errors from e.g. permission errors? The [API reference](https://cloud.google.com/resource-manager/docs/core_errors#Global_Errors) suggests that the API utilizes multiple error types that might help differentiate, but I'm not sure what your tests against the real API showed.

Reply:
Added the error check